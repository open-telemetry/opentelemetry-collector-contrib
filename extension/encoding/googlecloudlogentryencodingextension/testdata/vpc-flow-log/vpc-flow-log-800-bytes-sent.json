{
  "insertId": "uuq3awf13s959",
  "jsonPayload": {
    "connection": {
      "dest_ip": "192.0.2.3",
      "src_ip": "203.0.113.3",
      "protocol": 1
    },
    "start_time": "2025-09-27T21:11:25.762594203Z",
    "dest_instance": {
      "region": "asia-south1",
      "project_id": "test-project-id",
      "zone": "asia-south1-c",
      "vm_name": "test-vm-3"
    },
    "bytes_sent": "800",
    "end_time": "2025-09-27T21:11:25.762594203Z",
    "src_location": {
      "region": "Virginia",
      "city": "Ashburn",
      "asn": 14618,
      "country": "usa",
      "continent": "America"
    },
    "dest_vpc": {
      "vpc_name": "default",
      "subnetwork_name": "default",
      "project_id": "test-project-id",
      "subnetwork_region": "asia-south1"
    },
    "reporter": "DEST",
    "packets_sent": "20"
  },
  "resource": {
    "type": "vpc_flow_logs_config",
    "labels": {
      "resource_container": "projects/123456789012",
      "location": "global",
      "name": "test-flow-logs-config"
    }
  },
  "timestamp": "2025-09-27T21:11:28.320127676Z",
  "labels": {
    "target_resource_name": "projects/123456789012/global/networks/default"
  },
  "logName": "projects/test-project-id/logs/networkmanagement.googleapis.com%2Fvpc_flows",
  "receiveTimestamp": "2025-09-27T21:11:28.320127676Z"
}
