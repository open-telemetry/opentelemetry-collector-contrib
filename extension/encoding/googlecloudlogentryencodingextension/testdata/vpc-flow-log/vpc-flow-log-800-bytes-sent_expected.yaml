resourceLogs:
  - resource:
      attributes:
        - key: gcp.project
          value:
            stringValue: test-project-id
        - key: cloud.resource_id
          value:
            stringValue: networkmanagement.googleapis.com%2Fvpc_flows
        - key: gcp.resource_type
          value:
            stringValue: vpc_flow_logs_config
        - key: gcp.label.name
          value:
            stringValue: "test-flow-logs-config"
        - key: gcp.label.location
          value:
            stringValue: global
        - key: gcp.label.resource_container
          value:
            stringValue: projects/123456789012
    scopeLogs:
      - logRecords:
          - attributes:
              - key: log.record.uid
                value:
                  stringValue: uuq3awf13s959
              - key: gcp.label.target_resource_name
                value:
                  stringValue: projects/123456789012/global/networks/default
              - key: network.protocol.name
                value:
                  stringValue: icmp
              - key: source.address
                value:
                  stringValue: 203.0.113.3
              - key: destination.address
                value:
                  stringValue: 192.0.2.3
              - key: gcp.vpc.flow.reporter
                value:
                  stringValue: DEST
              - key: gcp.vpc.flow.bytes_sent
                value:
                  intValue: "800"
              - key: gcp.vpc.flow.packets_sent
                value:
                  intValue: "20"
              - key: gcp.vpc.flow.start_time
                value:
                  stringValue: "2025-09-27T21:11:25.762594203Z"
              - key: gcp.vpc.flow.end_time
                value:
                  stringValue: "2025-09-27T21:11:25.762594203Z"
              - key: gcp.vpc.flow.destination.instance.project.id
                value:
                  stringValue: test-project-id
              - key: gcp.vpc.flow.destination.instance.vm.region
                value:
                  stringValue: asia-south1
              - key: gcp.vpc.flow.destination.instance.vm.name
                value:
                  stringValue: test-vm-3
              - key: gcp.vpc.flow.destination.instance.vm.zone
                value:
                  stringValue: asia-south1-c
              - key: gcp.vpc.flow.destination.project.id
                value:
                  stringValue: test-project-id
              - key: gcp.vpc.flow.destination.subnet.name
                value:
                  stringValue: default
              - key: gcp.vpc.flow.destination.subnet.region
                value:
                  stringValue: asia-south1
              - key: gcp.vpc.flow.destination.vpc.name
                value:
                  stringValue: default
              - key: gcp.vpc.flow.source.asn
                value:
                  intValue: "14618"
              - key: gcp.vpc.flow.source.geo.city
                value:
                  stringValue: Ashburn
              - key: gcp.vpc.flow.source.geo.continent
                value:
                  stringValue: America
              - key: gcp.vpc.flow.source.geo.country.iso_code.alpha3
                value:
                  stringValue: usa
              - key: gcp.vpc.flow.source.geo.region
                value:
                  stringValue: Virginia
            body: {}
            observedTimeUnixNano: "1759007488320127676"
            timeUnixNano: "1759007488320127676"
        scope:
          attributes:
            - key: encoding.format
              value:
                stringValue: gcp.vpcflow
